consensus/ethash/consensus.go

```diff
@@ -268,7 +268,8 @@ func (ethash *Ethash) verifyHeader(chain consensus.ChainHeaderReader, header, pa
 	}
 	// Verify the header's timestamp
 	if !uncle {
-		if header.Time > uint64(unixNow+allowedFutureBlockTimeSeconds) {
+		// XXX 'ethash.config.PowMode < ModeFake'
```

Comment:
What does `XXX`  mean? Should we remove it?

Reply:
removed as unnecessary.